piker/brokers/ib/feed.py

```diff
@@ -713,6 +771,98 @@ def normalize(
     return data
 
 
+@async_lifo_cache()
+async def get_mkt_info(
```

Comment:
I'm the least happy with this implementation (in terms of existing backends) just because as a legacy market broker ib tends to assume a source asset for every exchange..

Ideally we can work toward getting this more refined going forward particularly if we're going to be re-doing our tsdb subsys backend choice(s) and don't have to rely on the table keys in the existing replicated data sets..